src/raster/processing/dem/aspect.rs

```diff
@@ -0,0 +1,125 @@
+use super::options::common_dem_options;
+use crate::cpl::CslStringList;
+use crate::raster::processing::dem::DemSlopeAlg;
+use std::num::NonZeroUsize;
+
+/// Configuration options for [`aspect()`][super::aspect()].
+#[derive(Debug, Clone, Default)]
+pub struct AspectOptions {
+    input_band: Option<NonZeroUsize>,
+    compute_edges: bool,
+    output_format: Option<String>,
+    additional_options: CslStringList,
+    algorithm: Option<DemSlopeAlg>,
+    zero_for_flat: Option<bool>,
+    trigonometric: Option<bool>,
+}
+
+impl AspectOptions {
+    /// Create a DEM-aspect options set.
+    pub fn new() -> Self {
+        Default::default()
+    }
+
+    common_dem_options!();
+
+    /// Specify the slope computation algorithm.
+    pub fn with_algorithm(&mut self, algorithm: DemSlopeAlg) -> &mut Self {
```

Comment:
We don't have getters for these, but the common options have them.

Reply:
It was oversight, but we're removing getters, right?